libs/net/template/Models/Charts/Options/ChartOptionsModel.cs

```diff
@@ -36,6 +36,6 @@ public class ChartOptionsModel
     /// get/set -
     /// </summary>
     [JsonPropertyName("maintainAspectRatio")]
-    public string? MaintainAspectRatio { get; set; }
+    public bool? MaintainAspectRatio { get; set; }
```

Comment:
Deserialization was failing here because it was expecting a string instead of a boolean value.